File: projects-application/core/src/main/java/com/mycompany/myproject/core/ApplicationBoot.java
package com.mycompany.myproject.core;

import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ImportResource;

@Slf4j
@RequiredArgsConstructor
@SpringBootApplication
@ImportResource("classpath*:config/*.groovy")
public class ApplicationBoot {

	public static void main(String[] args) {
		SpringApplication.run(ApplicationBoot.class, args);
	}

}

File: projects-application/core/src/main/java/com/mycompany/myproject/core/ApplicationBoot.kt
package com.mycompany.myproject.core

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.support.beans

@SpringBootApplication
class ApplicationBoot

fun main(args: Array<String>) {
	runApplication<ApplicationBoot>(*args) {
		addInitializers(beans {
		})
	}
}
